pkgs/development/compilers/elm/packages/node/default.nix

```diff
@@ -6,6 +6,20 @@ let
     inherit pkgs nodejs;
     inherit (pkgs.stdenv.hostPlatform) system;
   };
+  ESBUILD_BINARY_PATH = lib.getExe (
+      pkgs.esbuild.override {
+        buildGoModule = args: pkgs.buildGoModule (args // rec {
+          version = "0.20.2";
+          src = pkgs.fetchFromGitHub {
+            owner = "evanw";
+            repo = "esbuild";
+            rev = "v${version}";
+            hash = "sha256-h/Vqwax4B4nehRP9TaYbdixAZdb1hx373dNxNHvDrtY=";
+          };
+          vendorHash = "sha256-+BfxCyg0KkDQpHt/wycy/8CTG6YBA/VJvJFhhzUnSiQ=";
+        });
+      }
+    );
```

Comment:
It seemed very version sensitive when we tried. Had to provide the exact version.

Reply:
uh that will be a bit painful to maintain. Maybe it would make sense to port these away from node2nix to `buildNpmPackage` right away to prevent potential regression with future bulk updates.

Reply:
Agree on it being painful. A slight condolence is the fact that Elm packages are usually quite slow to release new updates. 

I tried porting to `buildNpmPackage` but got lost. I'd need some hand-holding to get it done.

Reply:
Tried again today during NixCon Hack Day: https://github.com/NixOS/nixpkgs/pull/351767